Explorer/Assets/DCL/Character/CharacterMotion/Systems/HeadIKSystem.cs

```diff
@@ -64,7 +64,9 @@ public override void Initialize()
         protected override void Update(float t)
         {
             UpdateDebugValues();
-            UpdateIKQuery(World, t, in camera.GetCameraComponent(World));
+
+            if (!World.Has<InWorldCamera.ScreencaptureCamera.InWorldCamera>(camera))
```

Comment:
Is there a reason for not including the namespace at the top of the file?

Reply:
it is because namespace name is colliding with the component name. I will add Component suffix to fix it